config-reloader/processors/extract_plugins.go

```diff
@@ -39,7 +39,7 @@ func (p *expandPluginsState) Process(input fluentd.Fragment) (fluentd.Fragment,
 	}
 
 	f := func(d *fluentd.Directive, ctx *ProcessorContext) error {
-		if d.Name != "match" {
+		if d.Name != "match" && d.Name != "store" {
```

Comment:
Can you also add a test case? I recently had a bug 10 lines below in this file. It should be trivial to add a new test for the nested <store>

Reply:
@jvassev I've added a test for this, I based it off of one of the existing tests.